app/views/articles/show.html.erb

```diff
@@ -31,6 +31,41 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": "True",
+    "url": "<%= article_url(@article) %>",
+    "image": {
+      "@type": "ImageObject",
+      "url": "<%= article_social_image_url(@article) %>"
+    },
+    "publisher": {
+      "@context": "http://schema.org",
+      "@type": "Organization"
+    },
+    "logo": {
+      "@context": "http://schema.org",
+      "@type": "ImageObject",
+      "url": "<%= cloudinary(SiteConfig.logo_png, 192, "png") %>",
+      "width": "192",
+      "height": "192"
+    },
+    "name": "<%= ApplicationConfig["COMMUNITY_NAME"] %> Community",
+    "headline": "<%= @article.title %>",
+    "author": {
+      "@context": "http://schema.org",
+      "@type": "Person",
+      "url": "<%= user_url(@user) %>",
+      "name": "<%= @user.name %>"
+    },
+    "datePublished": "<%= @article.published_timestamp %>",
+    "dateModified": ""<%= @article.edited_at&.utc&.iso8601 %>"",
+    "articleBody": "<%= @article.processed_html.html_safe %>"
```

Comment:
I'm seeing conflicting info about whether `articleBody` should be included.

https://support.google.com/webmasters/thread/8926582?hl=en

Reply:
Good catch! After looking into it some more, I have removed `articleBody`. 😄